app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -231,6 +252,7 @@ data class MapInitialValues(
     val mapInitialCenter: GeoPoint = MapScreen.MAP_INITIAL_CENTER,
 )
 
+@SuppressLint("ClickableViewAccessibility")
```

Comment:
Question: what is this about ?

Reply:
Okay so:
- `setOnTouchListener` has checks for accessibility touch control
- Thus, it requests (by a Lint warning) that we perform a click in the Listener, for accessibility reasons
- Since our TouchListener doesn't want to perform a touch, and just want to check for zoomLevel, I suppressed this warning

Reply:
Thanks!